dspace-server-webapp/src/main/java/org/dspace/app/rest/submit/factory/impl/ResourcePolicyAddPatchOperation.java

```diff
@@ -35,7 +36,7 @@
  *
  * @author Luigi Andrea Pascarelli (luigiandrea.pascarelli at 4science.it)
  */
-public class ResourcePolicyAddPatchOperation extends AddPatchOperation<ResourcePolicyRest> {
+public class ResourcePolicyAddPatchOperation extends AddPatchOperation<UploadAccessConditionDTO> {
```

Comment:
Based on the changes in this PR, I'm realizing that this PatchOperation class (and **all** of the `ResourcePolicy*PatchOperation` classes) is improperly named.

One of the points of confusion I had with this PR was that I couldn't understand why "generic" Resource Policy PATCH operations would use a Bitstream-specific DTO.   After digging deeper (thanks to @abollini 's JavaDocs), I'm realizing this PATCH class is **not generic** (i.e. it's not handling a patch on the `/api/authz/resourcepolicies` endpoint).  It's also specific to Bitstreams.

Could we use this PR as an opportunity to correct the names of these PatchOperation Classes?  They probably should be named:
* `BitstreamResourcePolicyAddPatchOperation`
* `BitstreamResourcePolicyReplacePatchOperation`
* `BitstreamResourcePolicyRemovePatchOperation`